tests/regression_tests/example_routes/smoothed/fuel/checkerboard.json

```diff
@@ -22676,167 +22675,126 @@
+                        36.6424037097851,
+                        36.89582271358701,
+                        42.3522098011789,
+                        42.53446806056759,
+                        48.07189680338631,
+                        48.167984655140415,
+                        53.79800462285707,
+                        59.41684379358922,
+                        59.53678332427055,
+                        65.04094194859157,
+                        65.26882074515389,
+                        70.67255260859858,
+                        70.98984487187565,
+                        76.31397979829676,
+                        76.69567799049719,
+                        81.96760237969283,
+                        82.38225704317566,
+                        87.63590514210313,
+                        88.04564766614064,
+                        93.32151866356138,
+                        93.68205301992437,
+                        99.0272708324409,
+                        99.2878187324707,
+                        104.75625454596597,
+                        104.85943497190614,
+                        107.62997510458867
                     ],
                     "distance": [
                         0.0,
-                        76302.63081599193,
-                        80180.17768091091,
-                        229866.1245284547,
-                        239701.52182551718,
-                        384532.28184265137,
-                        398442.12032149814,
-                        540246.7603618806,
-                        556446.5207286028,
-                        696944.5073320201,
-                        713765.9135123453,
-                        854549.4390136747,
-                        870456.8001704792,
-                        1012974.8091839311,
-                        1026579.5699409976,
-                        1172124.2510086778,
-                        1182197.0635316968,
-                        1331893.3867271268,
-                        1337373.2113448104,
-                        1492171.8234587512,
-                        1646696.8948264816,
-                        1652764.8752888201,
-                        1801689.9732666507,
-                        1812788.0772041008,
-                        1957211.5716882187,
-                        1972132.2599491281,
-                        2113321.9103546618,
-                        2130698.0303563294,
-                        2270081.141075568,
-                        2288395.736116514,
-                        2427549.972582523,
-                        2445144.931945467,
-                        2585790.900969311,
-                        2600873.371063741,
-                        2744870.2650521733,
-                        2755515.606999688,
-                        2904861.483574079,
-                        2909011.225408434,
-                        2985242.430609112
-                    ],
-                    "speed": [
```

Comment:
Looks like the speed values along the route have been removed here and not replaced? I presume that's what's causing the KeyError in the regression tests.